src/wifi/AdafruitIO_AIRLIFT.h

```diff
@@ -93,6 +93,25 @@ class AdafruitIO_AIRLIFT : public AdafruitIO {
     if (_http)
       delete _http;
   }
+  
+  
+  /**************************************************************************/
+  /*!
+  @brief  Allows setting of the AirLift RGB led from the 
+  Adafruit IO AirLift Class.
+  @param  r
+	  Red value
```

Comment:
What are the limits of the `r` `g` `b` values? Is this value constrained (0->255?)